template.yaml

```diff
@@ -552,12 +587,17 @@ Resources:
         Format: '{ "apiId": "$context.apiId", "requestId": "$context.requestId", "requestTime": "$context.requestTime", "requestTimeEpoch": "$context.requestTimeEpoch", "httpMethod": "$context.httpMethod", "path": "$context.path", "status": "$context.status",  "error.message": "$context.error.message" }'
       StageName: Prod
       Auth:
+        AddDefaultAuthorizerToCorsPreflight: false
+        DefaultAuthorizer: CognitoAuthorizer
         Authorizers:
           CognitoAuthorizer:
+            AuthorizationScopes:
```

Comment:
Are we still able to have different authorisers for different lambdas?

Reply:
Do you mean different scopes? The old solution had 1 Authoriser defined in customer-swagger which was not actually used. 
Can we do a huddle on this?